astropy/cosmology/tests/test_cosmology.py

```diff
@@ -1577,6 +1577,21 @@ def test_z_at_value():
     assert allclose(z_at_value(cosmo.distmod, 46 * u.mag),
                     1.9913891680278133, rtol=1e-6)
 
+    # Test array:
+    assert allclose(z_at_value(cosmo.distmod, np.array([46]*4) * u.mag,
+                               zmin=1.5, zmax=2.5, nbins=10000),
+                    1.9913891680278133, rtol=1e-3)
```

Comment:
Why is the tolerance only 1e-3? I thought ``z_at_value`` had a tolerance of 1e-8?
Same question for the following tests.

Reply:
But this is for the interpolation method – I'd also think it should be much better than 1e-3 with 10000 grid points though (`z_at_value` itself may have anything between some 1e-4 for restricted no. of iterations, and < 1e-11 for full roundtrip tests, see extended tests in 11080).